compiler/damlc/lib/DA/Cli/Damlc/DependencyDb.hs

```diff
@@ -110,8 +111,14 @@ installDependencies ::
    -> IO ()
 installDependencies projRoot opts sdkVer@(PackageSdkVersion thisSdkVer) pDeps pDataDeps = do
     deps <- expandSdkPackages (optDamlLfVersion opts) (filter (`notElem` basePackages) pDeps)
+    DataDeps {dataDepsDars, dataDepsDalfs, dataDepsPkgIds} <- readDataDeps pDataDeps
     (needsUpdate, newFingerprint) <-
-        depsNeedUpdate depsDir (deps ++ pDataDeps) thisSdkVer (show $ optDamlLfVersion opts)
+        depsNeedUpdate
+            depsDir
+            (deps ++ dataDepsDalfs)
```

Comment:
What about `dataDepsDars`? Don’t we need to include them in the check as well?

Reply:
that's a good catch, fixed.